### DISCUSSION TITLE
Decap Professional Help

### DISCUSSION THREAD

**Discussion by martinjagodic** [decaporg/decap-cms#7027] [Announcements]
We now offer professional services on top of Decap CMS. This is intended for users who are looking for expert guidance and ways to save time. We offer:
- Onboarding
- Monthly Support
- Development of Custom Features 

The services are described in detail here: https://techhub.p-m.si/services/decap/

We offer this because the questions in our help channel are often without answers, and I believe many questions are not even asked. So we want to provide a guaranteed answer service. Something similar can be said about feature requests, so now we provide a way to get yours done.

We are an independent team, so any order of such service contributes to the future of Decap.

This does not affect the open-source core of Decap CMS which will forever stay that way.

**Comment by pktiuk** [comment]
Maybe you should also consider option to offer bounties for dealing with some issues?

There are platforms for dealing with this like https://polar.sh/ or https://dev.algora.io/     
People could place bounties on selected issues to vote with wallets on some features.

**Reply by martinjagodic** [reply]
Thanks for the suggestion. We did think about it back then but didn't implement it. Maybe it's time to reconsider :)
